include/reactor-uc/clock_synchronization.h

```diff
@@ -55,7 +56,7 @@ struct ClockSynchronization {
   interval_t period;              // The period between sync request messages are sent to the neighbor master.
   ClockSyncTimestamps timestamps; // The timestamps used to compute clock offset.
   ClockServo servo;               // The PID controller
-  // TODO: Is this message here needed? Why cant we use the one on the bundle?
+  // FIXME: Is this message here needed? Why cant we use the one on the bundle?
```

Comment:
Depends. Currently, we don't have a clear policy where the FederatedMessage struct should life. 

At some places you use the FederatedMessage struct from the Channel, sometimes you have your own.